include/industry_standard/pcidoe.h

```diff
@@ -38,11 +37,9 @@ typedef struct {
 #define PCI_DOE_DATA_OBJECT_TYPE_SECURED_SPDM 0x02
 
 #define PCI_DOE_MAX_SIZE_IN_BYTE 0x00100000
-#define PCI_DOE_MAX_SIZE_IN_DW 0x00040000
-
-
-/* DOE Discovery*/
+#define PCI_DOE_MAX_SIZE_IN_DW (4 * PCI_DOE_MAX_SIZE_IN_BYTE)
```

Comment:
This should be `(PCI_DOE_MAX_SIZE_IN_BYTE / 4)` ?

Reply:
Yes.